tests/test-cases/Group10-VCH-Restart/10-01-VCH-Restart.robot

```diff
@@ -131,3 +131,22 @@ Create VCH attach disk and reboot
     Wait Until Keyword Succeeds  20x  5 seconds  Run Docker Info  %{VCH-PARAMS}
     ${rc}=  Run And Return Rc  govc device.ls -vm=%{VCH-NAME} | grep disk
     Should Be Equal As Integers  ${rc}  1
+
+Docker inspect mount data after reboot
+    ${rc}  ${container}=  Run And Return Rc And Output  docker %{VCH-PARAMS} volume create --name=named-volume
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${container}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create --name=mount-data-test -v /mnt/test -v named-volume:/mnt/named busybox
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${out}=  Run And Return Rc And Output  docker %{VCH-PARAMS} inspect -f '{{.Mounts}}' mount-data-test
```

Comment:
currently there is another issue with `inspect` where the `mountsFromContainer` function returns a different UUID for an anonymous volume each time(this is a bug). Because of this the output for anonymous volumes may be incorrect everytime inspect is called. You might check for this after your current PR. The behavior is likely the same and will need a separate fix to persist the appropriate information.